lib/scalarm/database/model/grid_credentials.rb

```diff
@@ -23,6 +23,8 @@ class GridCredentials < Scalarm::Database::MongoActiveRecord
     use_collection 'grid_credentials'
     attr_join :user, ScalarmUser
 
+    create_index "user_id"
```

Comment:
@dkrol - drobna uwaga co do spójności stylu (użycie ``"`` zamiast ``'``); to występuje także później. Tak samo w niektórych miejscach jest używany Hash. Ale to może do dyskusji przy okazji komentarza do ``create_index``.